src/Classes/ImportTab.lua

```diff
@@ -170,48 +170,106 @@ You can get this from your web browser's cookies while logged into the Path of E
 	self.controls.generateCodeCopy.enabled = function()
 		return #self.controls.generateCodeOut.buf > 0
 	end
-	self.controls.generateCodePastebin = new("ButtonControl", {"LEFT",self.controls.generateCodeCopy,"RIGHT"}, 8, 0, 140, 20, "Share with Pastebin", function()
-		local id = LaunchSubScript([[
-			local code, proxyURL = ...
-			local curl = require("lcurl.safe")
-			local page = ""
-			local easy = curl.easy()
-			easy:setopt_url("https://pastebin.com/api/api_post.php")
-			easy:setopt(curl.OPT_POST, true)
-			easy:setopt(curl.OPT_POSTFIELDS, "api_dev_key=c4757f22e50e65e21c53892fd8e0a9ff&api_paste_private=1&api_option=paste&api_paste_code="..code)
-			easy:setopt(curl.OPT_ACCEPT_ENCODING, "")
-			if proxyURL then
-				easy:setopt(curl.OPT_PROXY, proxyURL)
-			end
-			easy:setopt_writefunction(function(data)
-				page = page..data
-				return true
-			end)
-			easy:perform()
-			easy:close()
-			if page:match("pastebin.com") then
-				return page
-			else
-				return nil, page
+
+	-- For new export website, add another entry to this table with data filled out
+	-- Label for dropdown, id for err, subScript for POST call, codeOut for SetText
+	-- codeOut if we need to append anything (*in front of) response, pastebin returns full link, pobbin returns only id for example
+	local exportWebsiteList = {
+		{
+			label = "Pastebin.com",
+			id = "Pastebin",
+			codeOut = "",
```

Comment:
Shouldn't this have the pastebin link here?

Reply:
Pastebin returns the full link in the response so we just output that to the generateCodeOut. We could actually ask @Dav1dde about returning the same way (full https link with id) and we could completely remove the need for codeOut here and down below in the Share button function. But I did JUST ask him to change it to id for me and I like to think there's a cooldown on requests😄

Reply:
Gotcha, didn't realize that it returns like that, just reading through code right now, not testing functionality.  This is fine, then, makes it more flexible